python/ray/experimental/slib/examples/simple.py

```diff
@@ -0,0 +1,84 @@
+import time
+import logging
+import argparse
+
+from ray.experimental.slib.streaming import *
+
+logger = logging.getLogger(__name__)
+logger.setLevel("INFO")
+
+# Test functions
+def splitter(line):
+    return line.split(' ')
+
+def filter_fn(word):
+     if "f" in word: return True
+     return False
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--input-file", required=True,
+                    help="the input text file")
+
+if __name__ == '__main__':
```

Comment:
```suggestion
if __name__ == "__main__":
```

We use `"` throughout the codebase